src/Octokit.Webhooks/Extensions/StringEnum.cs

```diff
@@ -101,4 +149,14 @@ private static TEnum ToEnum(string str)
 
         throw new ArgumentException(str);
     }
+
+    private TEnum ParseValue()
+    {
+        if (this.TryParse(out var value))
```

Comment:
Would it be beneficial to add nullable annotations like NotNullWhen to the TryParse method?